plugins/inputs/lustre2/lustre2_test.go

```diff
@@ -16,6 +16,49 @@ import (
 
 // Set config file variables to point to fake directory structure instead of /proc?
 
+// Subset of a brw_stats file. Contains all headers, with representative buckets.
+const brwstatsProcContents = `snapshot_time:         1589909588.327213269 (secs.nsecs)
+
+                           read      |     write
+pages per bulk r/w     rpcs  % cum % |  rpcs        % cum %
+1:                    5271   0   0   | 337023  22  22
+2:                    3030   0   0   | 5672   0  23
+4:                    4449   0   0   | 255983  17  40
+8:                    2780   0   0   | 33612   2  42
+
+                           read      |     write
+discontiguous pages    rpcs  % cum % |  rpcs        % cum %
+0:                43942683 100 100   | 337023  22  22
+1:                       0   0 100   | 5672   0  23
+2:                       0   0 100   | 28016   1  24
+3:                       0   0 100   | 227967  15  40
+4:                       0   0 100   | 12869   0  41
+
+                           read      |     write
+disk I/Os in flight    ios   % cum % |  ios         % cum %
+1:                 2892221   6   6   | 1437946  96  96
+2:                 2763141   6  12   | 44373   2  99
+3:                 3014304   6  19   | 2677   0  99
+4:                 3212360   7  27   |  183   0  99
+
+                           read      |     write
+I/O time (1/1000s)     ios   % cum % |  ios         % cum %
+1:                  521780   1   1   |    0   0   0
+16:                6035560  16  22   |    0   0   0
+128:               5044958  14  98   |    0   0   0
+1K:                    651   0  99   |    0   0   0
+
+                           read      |     write
+disk I/O size          ios   % cum % |  ios         % cum %
+1:                       0   0   0   | 327301  22  22
+16:                      0   0   0   |    0   0  22
+128:                    35   0   0   |  209   0  22
+1K:                      0   0   0   | 1703   0  22
+16K:                  4449   0   0   | 255983  17  40
+128K:                  855   0   0   |   23   0  42
+1M:               43866371  99 100   | 850248  57 100
+`
+
```

Comment:
Can you please move this to a file in a `testcases` or `testdata` subdir and load it where needed? You can even reproduce the `proc` directory structure there if you need to...